backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc

```diff
@@ -104,17 +104,25 @@ EXTERNAL_DNS_DOMAIN="hc.jpdv.aws.kerbeross.com"
 
 The backup and restore process works as follows:
 
-. On management cluster 1, which you can think of as the source management cluster, the control plane and workers interact by using the ExternalDNS API.
+. On management cluster 1, which you can think of as the source management cluster, the control plane and workers interact by using the external DNS API. The external DNS API is accessible, and a load balancer sits between the management clusters.
++
+image::298_OpenShift_Backup_Restore_0123_00.png[Diagram that shows the workers accessing the external DNS API and the external DNS API pointing to the control plane through a load balancer]
 
-. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes are moved to the external DNS, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket.
+. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes continue to try to access the external DNS API even if it is not accessible, the workloads are running, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket. The data plane is active and the control plane is paused.
```

Comment:
The _The worker nodes continue to try to access_ sentence confuses me a bit. It sounds like the worker nodes will continue if the following 4 conditions exist. I don't think that is true, though. Would splitting the sentence after _it is not accessible_ and re-wording the rest slightly to make a second sentence be right?

Reply:
Oooh. I like the _During this process_ in the next step. Would that work here?

Reply:
I'll try it!

Reply:
@mburke5678 I incorporated your suggestion. Thanks!